midp/gfx.js

```diff
@@ -896,74 +896,31 @@ var currentlyFocusedTextEditor;
     }
 
     function setClip(g, x, y, width, height) {
```

Comment:
This will also improve performance, because in many natives we don't need to call save() if the graphics element isn't clipped!